docs/serving/creating-domain-mappings.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Creating a Mapping between a Custom Domain Name and a Knative Service (Alpha)"
+linkTitle: "Creating Domain Mappings (Alpha)"
+weight: 64
+type: "docs"
+---
```

Comment:
The intro needs some use case if not would very confusing for the end user on why she would need this feature. 

You can put something like 
knative by default offeres a comfigmap “config-domain”  in the control plane namespace  (Ie knative-serving”) that allow an admin with write access to the namespace edit the comfigmap  to configure domain names including a selector to be use which services corresponds to a domain name.. 
But in some cases the admin would not want this ConfigMap to be edited and would prefer users to define the domain name mapping to be fine by users in the namespace where the knative service is deploy. 
This feature would allow to define the mapping in their namespace by creating a Kubernetes resource DomainMapping

Reply:
How about just "Knative Services are automatically given a default domain name based on the cluster configuration. You can also map a single custom domain name that you own to a specific Knative Service using the Domain Mapping feature."

Reply:
sure